Gemfile

```diff
@@ -2,3 +2,7 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in trix.gemspec
 gemspec
+
+group :test do
+  gem 'test-unit'
```

Comment:
The test threw some missing dependency error, and this line solved it, but I removed it now and everything is still working as charm. I tested it on a freshly installed linux as well. Something was fishy...